src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -764,6 +777,16 @@ public function getFormConfig()
         return $config;
     }
 
+    /**
+     * @param callable(mixed $modelData) $isEmptyCallback
```

Comment:
we can use real type declarations instead

Reply:
Alright. I intentionally did not strict type it to callable to allow array and string callables but we can force it.

Reply:
This still works: https://3v4l.org/UkNft

Reply:
🤔 Ok. I definitely did it like this firstly but my tests had types errors. I'm gonna check it again. Thank you!